lib/src/painters/solid_path_painter.dart

```diff
@@ -4,21 +4,33 @@ import 'package:flutter/material.dart';
 class SolidCropPathPainter extends CustomPainter {
   static const _strokeWidth = 4.0;
   final Path _path;
-  final _paint = Paint()
-    ..color = Colors.white
-    ..strokeWidth = _strokeWidth
-    ..style = PaintingStyle.stroke
-    ..strokeJoin = StrokeJoin.round;
+  final Paint pathPaint;
 
   /// Draw a solid path around the given path
-  SolidCropPathPainter(this._path);
+  SolidCropPathPainter(this._path, {required this.pathPaint});
```

Comment:
best to keep them the same, both named or both positioned. My preference goes to names, but either is fine. Just not one positioned and one named

Reply:
Done.